docs/dotnet/dotnet-aspnet-metrics.md

```diff
@@ -0,0 +1,216 @@
+# Semantic Conventions for ASP.NEt Core specific metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for ASP.NET Core metrics, not specific to HTTP
+
+**Disclaimer:** These are initial .NET metric instruments available in .NET 8 but more may be added in the future.
+
+<!-- toc -->
+
+- [Routing](#routing)
+  * [Metric: `aspnet.routing.successful_matches`](#metric-aspnetroutingsuccessful_matches)
+  * [Metric: `aspnet.routing.failed_matches`](#metric-aspnetroutingfailed_matches)
+- [Exception metrics](#exception-metrics)
+  * [Metric: `aspnet.diagnostics_handler.exceptions`](#metric-aspnetdiagnostics_handlerexceptions)
+- [Rate-limiting](#rate-limiting)
+  * [Metric: `aspnet.rate_limiting.active_request_leases`](#metric-aspnetrate_limitingactive_request_leases)
+  * [Metric: `aspnet.rate_limiting.request_lease.duration`](#metric-aspnetrate_limitingrequest_leaseduration)
+  * [Metric: `aspnet.rate_limiting.queued_requests`](#metric-aspnetrate_limitingqueued_requests)
+  * [Metric: `aspnet.rate_limiting.queued_requests.duration`](#metric-aspnetrate_limitingqueued_requestsduration)
+  * [Metric: `aspnet.rate_limiting.rejected_requests`](#metric-aspnetrate_limitingrejected_requests)
+
+<!-- tocstop -->
+
+## Routing
+
+All routing metrics are reported by `Microsoft.AspNetCore.Routing` meter.
+
+### Metric: `aspnet.routing.successful_matches`
+
+**Status**: [Experimental][DocumentStatus]
+
+**TODO: can we try one aspnet.routing.matches** ?
```

Comment:
No objection from me and James' comment below didn't sound like an objection. I'd say lets propose it that way and we'll see if anyone pushes back.

Reply:
resolving in favor of https://github.com/lmolkova/semantic-conventions/pull/1#discussion_r1271241306